docs/operator/configuring-eventing-cr.md

```diff
@@ -0,0 +1,117 @@
+---
+title: "Configuring Eventing Operator Custom Resource"
+weight: 10
+type: "docs"
+aliases:
+- /docs/operator/configuring-eventing-cr/
+---
+
+# Configuring Eventing Operator Custom Resource
+
+You are only able to configure Knative Eventing with the following option:
+
+* Private repository and private secret
+
+We configure Eventing operator CR the same way as Serving operator CR, with the same fields. Refer to the section of
+“[Private repository and private secret](configuring-serving-cr.md#private-repository-and-private-secrets)” in Serving operator for detailed instruction. The difference is that Knative
+Eventing Operator only allows us to customize the images for all deployments: `eventing-controller`, `eventing-webhook`,
+`imc-controller`, `imc-dispatcher`, and `broker-controller`. You need to either use these names as your names of the images
+in the repository, or to map your image links on a one-on-one basis.
+
+## Example 1 — download images in a predefined format without secrets:
+
+Suppose you use the custom tag v0.13.0 for all your images, and all the image links you have are able to accessible
+without secrets, and defined in the expected format: `docker.io/knative-images/${NAME}:v0.13.0`, you need to define your
+operator CR with following content:
+
+```
+apiVersion: operator.knative.dev/v1alpha1
+kind: KnativeEventing
+metadata:
+  name: knative-eventing
+  namespace: knative-eventing
+spec:
+  registry:
+    default: docker.io/knative-images/${NAME}:v0.13.0
+```
+
+With that said, you need to make sure your images are saved in the following link:
+
+* Image of `eventing-controller`: `docker.io/knative-images/eventing-controller:v0.13.0`.
+* Image of `eventing-webhook`: `docker.io/knative-images/eventing-webhook:v0.13.0`.
+* Image of `imc-controller`: `docker.io/knative-images/imc-controller:v0.13.0`.
+* Image of `imc-dispatcher`: `docker.io/knative-images/imc-dispatcher:v0.13.0`.
+* Image of `broker-controller`: `docker.io/knative-images/broker-controller:v0.13.0`.
+
+This is the simplest case to define your custom image links. Be aware that if you use this approach, you must make sure
+that all the images in Knative Eventing will be replaced with your own images.
+
+## Example 2 — download images from different repositories without secrets:
+
+If all the images are not saved in a uniformed format, we need to define them on a one-on-one basis.
+
+Suppose we would like to download the images as below:
```

Comment:
Instead:

"For example, to define the list of images:"
(list of images)
"The operator CR should be modified to include the full list:"